source/includes/rars_unique_domain.md

```diff
@@ -0,0 +1,46 @@
+## Unique Domain
+
+Returns a list of unique domains that are being tracked for a given advertiser in the past 14 days. The data returned is a list of unique domains along with corresponding visit counts.
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/unique-domain/[gmaid] |
+
+
+### Response Body
+
+The body of the API response is an array of unique domain objects.
+
+Field Name | Datatype | Nullable | Description
+---------- | -------- | -------- | -----------
+domain | String | no | The unique domain corresponding to the count
+count | Integer | no | The total visit counts to the specified domain in the last 14 days.
+
+
+```
+curl --request GET \
+  --url 'https://api.localiqservices.com/client_reports/unique-domain/TEST_1' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Bearer OAUTH_ACCESS_TOKEN'
+```
+
+> > Example Response
+
+```json
+[
+    {
+        "domain": "example.com",
+        "count": 80
+    },
+    {
+        "domain": "admin.example.com",
```

Comment:
This doesn't look like a RARS response.  For the other endpoints we took the liberty of making them fit with our schemes as they are new API endpoints.  See [top_referrers](https://github.com/GannettDigital/api-docs/blob/master/source/includes/_rars_top_referrers.md).  [I suspect that this isn't the response](https://github.com/GannettDigital/reach-analytics-reporting-service/pull/1722/files#diff-85a625e775ca8e46761006cc9b89ebe1b1bc6e9ecc25234b9210d11533d23423R15) due to the lack of `report_data` and I don't see anything obvious in your code PR changing the behavior.

Reply:
Good catch. I've made the update.